### ISSUE TITLE
Should I use pretained weights or let it learn by neural network?

### ISSUE THREAD

**Issue by Abhinav43** [benedekrozemberczki/graph2vec#10]
I have trained my graphs with graph2vec, Now i want to use in tensorflow model. Should I set trainable = False and use fixed pretrained vectors or should I set trainable = true and let it train?

Thanks

**Comment by benedekrozemberczki** [comment]
Why would you want to use the Tensorflow version?

**Comment by Abhinav43** [comment]
No, I am not using tensorflow version of this embeddings
, I am training with this version and then using the embedding in my tensorflow network ( which is another one )

**Comment by benedekrozemberczki** [comment]
I would set it to be non-trainable.

**Comment by benedekrozemberczki** [comment]
Also, a lower learning rate for those weights would help.

**Comment by Abhinav43** [comment]
Cool, thanks for quick reply.
